set-vars.md

```diff
@@ -0,0 +1,155 @@
+---
+title: SET (session settings)
+summary: The SET statement modifies the current settings for the client session.
+toc: false
+---
+
+The `SET` [statement](sql-statements.html) can modify one of the
+session setting variables. These can also be queried via [`SHOW`](show-vars.html).
+
+{{site.data.alerts.callout_danger}}In some cases, client drivers can drop and restart the connection to the server. When this happens, any session configurations made with <code>SET</code> statements are lost. It is therefore more reliable to configure the session in the client's connection string. For examples in different languages, see the <a href="build-an-app-with-cockroachdb.html">Build an App with CockroachDB</a> tutorials.{{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to modify the session settings.
+
+## Synopsis
+
+General syntax:
+
+{% include sql/diagrams/set_var.html %}
+
+The variable name is case insensitive.
+
+Special syntax cases:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SET SESSION CHARACTERISTICS AS TRANSACTION ISOLATION LEVEL ...` | `SET default_transaction_isolation = ...`
+| `SET TIME ZONE ...` | Special syntax because the variable name contains a space. See [`SET TIME ZONE`](#set-time-zone) below.
+
+{{site.data.alerts.callout_info}}The <code>SET</code> statement for session settings is unrelated to the other <a href="set-transaction.html"><code>SET TRANSACTION</code></a> and <a href="cluster-settings.html#change-a-cluster-setting"><code>SET CLUSTER SETTING</code><a> statements.{{site.data.alerts.end}}
```

Comment:
The last `<a>` tag needs to be `</a>`. As is, it's messing up the styling of the table below.